google_symptoms/DETAILS.md

```diff
@@ -0,0 +1,26 @@
+# Google Symptoms
+
+We import the confirmed case and deaths data from the Google Research's
+Open COVID-19 Data project and export the county-level and state-level data
```

Comment:
How important is it to have this signal available at other geo types (msa, hrr) @krivard ?

Values aren't directly comparable across geo regions due to the normalization process. Search volume is normalized by population of the region and by the highest symptom search volume in a region at some set point in time. So, effectively, we have symptom search terms ranked by popularity. It's not obvious how to aggregate this from counties up to other geo types.

Maybe to aggregate, we could average search term popularity from member regions, weighting by population, which would produce an average per-capita search term rank.

Reply:
Our map also shows MSA, so if we put the symptoms signal on the map, it would be missing from that level. It's not the end of the world, though, so if aggregating takes more work, we could let it wait.

(And maybe we have a contact from google who can tell us the best aggregation strategy, if we choose to do so?)

Reply:
We do have a few folks at Google who are our primary contacts for Google Symptoms.  @jsharpna do you think this is something appropriate to bring to them (and in general, what do you think of using population-weighted averaging as an aggregation scheme)?

My two cents: I think population-weighted averaging is totally reasonable, since the signal is already on the same scale for each county.  Also, population-weighted averaging was the default for aggregating such signals in March/April - it's intuitive, preserves the scale, etc...

The ideal would be to obtain MSA-level signals directly from Google.  (As we discussed >1 month ago, this is the kind of ask from Google that would be reasonable (along with, e.g., obtaining "union / intersection" signals), but we shouldn't make such an ask empty handed (i.e., we should demonstrate the usefulness of these additional features to motivate the additional work that we're asking them to do for us).